yearn/contracts/ConvexStrategy.sol

```diff
@@ -154,6 +158,9 @@ contract ConvexStrategy is BaseStrategy {
     // Address of the WBTC token contract.
     address public constant wbtc =
         address(0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599);
+    // Address of the tBTC token contract.
+    address public constant tbtc =
```

Comment:
I totally agree on adding TODOs but not sure about replacing the current approach by programmatically fetching tbtc(v1/v2) address. Once the tBTCv2 is deployed it stays there and I think we can safely hardcode it just like with other tokens. Adding code for fetching the address surely works, but it adds some code that can be avoided I think. Regarding the index, similar thoughts as for the token address. Once the pool is deployed it stays there and the indexes won't change just like we do for `wBTC` index. I guess we can assume the index will be `0` for tBTC but we also don't know for sure if Curve has any plans with updating their pool contracts and how the indexes will look like. Idk, I think the safest and the most simple way is to wait until all the contracts are deployed (tBTCv2 + curve pool with tBTCv2) and use the addresses and indexes accordingly.
If you feel strongly about fetching addresses programatically and assuming the index will still be `0`, I guess we can do that as well. 
Added TODOs here 45e577f

Reply:
Yeah those arguments sound reasonable. But, what do you think about using wBTC instead of tBTC as we planned at the beginning? This way we don't have to depend on tBTC address and we can rely on wBTC which is stable and already used in those contracts. The only concern we should remember about is checking if wBTC index in the v2 pool remains the same as in the current v1 pool. We can leave TODOs for that here (near `calc_token_amount` invocation) and next to `add_liquidity` calls in both strategies. What do you think?

Reply:
Yes! I like this idea, we can simplify existing approach by replacing `tBTC` with `wBTC`